src/main/kotlin/no/nav/helsearbeidsgiver/bro/sykepenger/kafkatopic/pri/Pri.kt

```diff
@@ -26,6 +26,8 @@ object Pri {
         SKAL_HA_PAAMINNELSE("skal_ha_paaminnelse"),
         FORESPOERSEL("forespoersel"),
         EKSPONERT_FORESPOERSEL_ID("eksponertForespoerselId"),
+        VEDTAKSPERIODE_ID("vedtaksperiodeId"),
```

Comment:
Helst bruk snake_case i strengen for nye nøkler. camelCase er et gammelt feilsteg.